devops/openshift/queue-consumers-deploy.yml

```diff
@@ -261,13 +261,30 @@ objects:
   - apiVersion: route.openshift.io/v1
     kind: Route
     metadata:
-      name: ${NAME}
+      name: ${NAME}-route
+      annotations:
+        haproxy.router.openshift.io/balance: leastconn
+        haproxy.router.openshift.io/disable_cookies: "true"
+        haproxy.router.openshift.io/hsts_header: max-age=31536000;includeSubDomains;preload
+        haproxy.router.openshift.io/timeout: 30s
     spec:
       host: ${HOST_NAME}
       path: ${PATH}
       tls:
         insecureEdgeTerminationPolicy: Redirect
         termination: edge
+        certificate: |
+          -----BEGIN CERTIFICATE-----
+          ${TLS_CERTIFICATE}
+          -----END CERTIFICATE-----
+        key: |
+          -----BEGIN PRIVATE KEY-----
+          ${TLS_KEY}
+          -----END PRIVATE KEY-----
+        caCertificate: |
```

Comment:
Is it not `destinationCACertificate`?? 

![image](https://github.com/user-attachments/assets/b0d0b5d7-23f1-4c71-8d2f-2e530a46b531)

Reply:
The expected one is `caCertificate`. The [route](https://console.apps.silver.devops.gov.bc.ca/k8s/ns/0c27fb-dev/routes/tls-dev-edwapi/yaml) in the ticket can be used as an example.

Reply:
Ah!, I see now and thanks for explaining.